py/examples/tour.py

```diff
@@ -29,24 +29,45 @@ def __init__(self, filename: str, title: str, description: str, source: str):
         self.code = highlight(source, py_lexer, html_formatter)
         self.previous_example: Optional[Example] = None
         self.next_example: Optional[Example] = None
-        self.process: Optional[asyncio.subprocess.Process] = None
+        self.process: Optional[subprocess.Popen] = None
         self.is_app = source.find('@app(') > 0
 
     async def start(self):
         if self.is_app:
-            self.process = await asyncio.create_subprocess_exec(
-                sys.executable, '-m', 'uvicorn', '--port', _app_port, f'examples.{self.name}:main', env=dict(
-                    H2O_WAVE_EXTERNAL_ADDRESS=f'http://{_app_host}:{_app_port}'
-                ),
-            )
+            self.process = subprocess.Popen(
+                [sys.executable, '-m', 'uvicorn', '--port', _app_port, f'examples.{self.name}:main'],
+                env=self.adjust_env_for_platform(dict(H2O_WAVE_EXTERNAL_ADDRESS=f'http://{_app_host}:{_app_port}')))
```

Comment:
If I'm not mistaken, all this is doing is:
"if the platform is Windows and `SYSTEMROOT` is present in the OS env, copy it to `env`"

If so, it's simpler to inline it here instead of additional methods - the methods are static anyway.

Also, it would be good to add a comment why `SYSTEMROOT` needs to be copied to `env`.